extension/popup.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css?family=Lato');
+
+@font-face {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 300;
+  src: local('Lato Light'), local('fonts/Lato.woff2'), url(https://fonts.googleapis.com/css?family=Lato) format('woff2');
+}
+
+@font-face {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Lato Bold'), local('fonts/Lato.woff2'), url(https://fonts.googleapis.com/css?family=Lato) format('woff2');
+}
+
+body {
+    font-family: "Lato", sans-serif;
+    margin: 0px;
+    padding: 0px;
+    text-align: left;
+    width:220px;
+}
+
+div {
+    background-color:#fff;
+
+}
+
+form {
+  display: inline-block;
```

Comment:
Using `display: inline-block;` here seems to cause some pain when you change the `body`s `width` (this element's width doesn't fill). I would change it to `display: flex;` which will allow you to use `align-items: center;` to vertically align the `<input>` and `<i>` together inside the `<form>` (try it and watch the positioning, if that doesn't make sense). It will also freely fit to the width of its parent, meaning when you change the width on `<body>`, the `<form>` element here should also conform.